2023-02-04 21:01:20,064 INFO    MainThread:22631 [internal.py:wandb_internal():89] W&B internal server running at pid: 22631, started at: 2023-02-04 21:01:20.063199
2023-02-04 21:01:20,071 DEBUG   HandlerThread:22631 [handler.py:handle_request():126] handle_request: run_start
2023-02-04 21:01:20,072 INFO    WriterThread:22631 [datastore.py:open_for_write():77] open: /home/bhavyajeet.singh/msme/MEMS-XF2T/generation/mT5-baseline/wandb/offline-run-20230204_210118-3g33fzfh/run-3g33fzfh.wandb
2023-02-04 21:01:24,656 DEBUG   HandlerThread:22631 [meta.py:__init__():40] meta init
2023-02-04 21:01:24,657 DEBUG   HandlerThread:22631 [meta.py:__init__():54] meta init done
2023-02-04 21:01:24,657 DEBUG   HandlerThread:22631 [meta.py:probe():211] probe
2023-02-04 21:01:24,666 DEBUG   HandlerThread:22631 [meta.py:_setup_git():201] setup git
2023-02-04 21:01:24,711 DEBUG   HandlerThread:22631 [meta.py:_setup_git():208] setup git done
2023-02-04 21:01:24,711 DEBUG   HandlerThread:22631 [meta.py:_save_pip():58] save pip
2023-02-04 21:01:24,713 DEBUG   HandlerThread:22631 [meta.py:_save_pip():72] save pip done
2023-02-04 21:01:24,713 DEBUG   HandlerThread:22631 [meta.py:_save_conda():79] save conda
2023-02-04 21:01:37,247 DEBUG   HandlerThread:22631 [meta.py:_save_conda():87] save conda done
2023-02-04 21:01:37,247 DEBUG   HandlerThread:22631 [meta.py:probe():249] probe done
2023-02-04 21:01:46,137 WARNING MainThread:22631 [internal.py:wandb_internal():145] Internal process interrupt: 1
2023-02-04 21:01:46,841 WARNING MainThread:22631 [internal.py:wandb_internal():145] Internal process interrupt: 2
2023-02-04 21:01:46,842 ERROR   MainThread:22631 [internal.py:wandb_internal():148] Internal process interrupted.
2023-02-04 21:01:47,087 INFO    SenderThread:22631 [sender.py:finish():986] shutting down sender
2023-02-04 21:01:47,126 INFO    WriterThread:22631 [datastore.py:close():281] close: /home/bhavyajeet.singh/msme/MEMS-XF2T/generation/mT5-baseline/wandb/offline-run-20230204_210118-3g33fzfh/run-3g33fzfh.wandb
2023-02-04 21:01:47,128 INFO    HandlerThread:22631 [handler.py:finish():683] shutting down handler
2023-02-04 21:01:47,132 INFO    MainThread:22631 [internal.py:handle_exit():76] Internal process exited
